### PR TITLE
Testing for Book and Category

### PR DESCRIPTION
**Pull request by holsoni**
image

### CODE REVIEW

File: src/test/java/com/example/bookstore/controller/BookControllerTest.java [hunk 1]
```diff
@@ -0,0 +1,274 @@
...
+            new CreateBookRequestDto()
+                    .setTitle("Harry Potter")
+                    .setAuthor("Rowling Joan")
+                    .setIsbn("978-3-16-148410-0")
+                    .setPrice(BigDecimal.valueOf(50.00))
+                    .setDescription("Very cool book")
+                    .setCoverImage("cover.jpeg")
+                    .setCategoryIds(Set.of(VALID_ID));
+
+    private static final CreateBookRequestDto INVALID_CREATE_BOOK_REQUEST_DTO =
```
Comment (Someboty) [inline src/test/java/com/example/bookstore/controller/BookControllerTest.java]: Bad naming

File: src/test/resources/database/scripts/delete-all-from-book_category.sql [hunk 2]
```diff
@@ -0,0 +1 @@
+DELETE FROM book_category;
```
Comment (Someboty) [inline src/test/resources/database/scripts/delete-all-from-book_category.sql]: Forgot empty line

File: src/test/resources/database/scripts/delete-all-from-books.sql [hunk 3]
```diff
@@ -0,0 +1 @@
+DELETE FROM book;
```
Comment (Someboty) [inline src/test/resources/database/scripts/delete-all-from-books.sql]: And here

File: src/test/resources/database/scripts/delete-all-from-category.sql [hunk 4]
```diff
@@ -0,0 +1 @@
+DELETE FROM category;
```
Comment (Someboty) [inline src/test/resources/database/scripts/delete-all-from-category.sql]: And here

File: src/test/resources/database/scripts/insert-book-category-relations-for-harry_potter.sql [hunk 5]
```diff
@@ -0,0 +1,2 @@
+insert into book_category (category_id, book_id)
+VALUES (1, 1);
```
Comment (Someboty) [inline src/test/resources/database/scripts/insert-book-category-relations-for-harry_potter.sql]: And here

**Review by Someboty — CHANGES_REQUESTED** [review]
GJ! Left few suggestions.

**Review by vovasalyha — APPROVED** [review]
GJ

**Pull request merged**